script/export_for_foi/export_hosts.sql

```diff
@@ -0,0 +1,8 @@
+COPY (
+  SELECT
+    sites.abbr AS "Abbreviation",
+    hostname
+    FROM hosts
+    INNER JOIN sites ON hosts.site_id = sites.id
+    ORDER BY sites.abbr, hostname
```

Comment:
I thought about this but settled on including them as they are (generally) working domains and some partial transitions use them to serve real users. On second thoughts though, you are right.